app/views/application/_user.html.erb

```diff
@@ -0,0 +1,23 @@
+<div class="col-md-12">
+	<div class="media b-media">
+		<a href="#"><img class="d-flex" src="<%= user.photo%>"
+				alt="<%= user.photo%>" height="256" width="256"></a>
```

Comment:
- [ ] When I visit the post index page, I am faced with this error. Kindly look into it and fix it.

- [ ] Also ensure the page layout uses [the wireframe](https://github.com/microverseinc/curriculum-rails/raw/main/blog-app/images/blog_user_all_posts.png).

  ![image](https://github.com/Izebiafe/solo-blog/assets/81468675/096eea11-502d-40fd-bfa2-1e9ac42bbbc3)

Reply:
- [ ] As requested by the previous reviewer, kindly fix the error the user faces when trying to access the post-index page
![image](https://github.com/Izebiafe/solo-blog/assets/81932478/ebaeacb7-2b9a-43f6-b920-c7ac35c213fe)